runbotics/runbotics-orchestrator-ui/src/views/process/ProcessBuildView/Modeler/ConfigureActionPanel/widgets/FieldTemplate/AdditionalPropertiesField.tsx

```diff
@@ -0,0 +1,110 @@
+import React, { useEffect } from 'react';
+
+import Remove from '@mui/icons-material/Remove';
+import { Button, TextField, Grid } from '@mui/material';
+import { utils } from '@rjsf/core';
+import i18n from 'i18next';
+import { JSONSchema7 } from 'json-schema';
+import dynamic from 'next/dynamic';
+
+import useTranslations from 'src/hooks/useTranslations';
+import { capitalizeFirstLetter } from 'src/utils/text';
+
+const ElementAwareAutocompleteWidget = dynamic(() => import('../ElementAwareAutocompleteWidget'), { ssr: false });
+
+const { ADDITIONAL_PROPERTY_FLAG } = utils;
+
+type JSONSchema7Titles = JSONSchema7 & { mainTitle?: string, subTitle?: string }; 
+
+type AdditionalPropertiesFieldProps = {
+    children: React.ReactElement;
+    // eslint-disable-next-line react/no-unused-prop-types
+    classNames: string;
+    disabled: boolean;
+    id: string;
+    label: string;
+    onDropPropertyClick: (index: string) => (event?: any) => void;
+    onKeyChange: (index: string) => (event?: any) => void;
+    readonly: boolean;
+    required: boolean;
+    schema: JSONSchema7Titles;
+};
+
+const AdditionalPropertiesField = ({
```

Comment:
if its arrow function component its better to use `: FC<AdditionalPropertiesFieldProps> = ({...}) =>`